mteb/benchmarks/benchmarks.py

```diff
@@ -188,33 +222,37 @@ def __getitem__(self, index):
 )
 
 MTEB_RETRIEVAL_LAW = Benchmark(
-    name="MTEB(law)",
-    tasks=[
-        "LegalSummarization",
-        "LegalBenchConsumerContractsQA",
-        "LegalBenchCorporateLobbying",
-        "AILACasedocs",
-        "AILAStatutes",
-        "LeCaRDv2",
-        "LegalQuAD",
-        "GerDaLIRSmall",
-    ],
-    description="Legal benchmarks from MTEB",
+    name="MTEB(law)",  # This benchmark is likely in the need of an update
+    tasks=get_tasks(
+        tasks=[
```

Comment:
Could this be sorted in alphabetical order?

Reply:
from the other questions I assume tasks